downloads/github-git-cheat-sheet-v1.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Git Cheat Sheet
+byline: Git is the open source distributed version control system that facilitates GitHub activities on your laptop or desktop. This cheat sheet summarizes commonly used Git command line instructions for quick reference.
+leadingpath: ../../
```

Comment:
This is the old cheat sheet, right @brianamarie? Since we have Git at our disposal and it lives in the history, I'd 👍 with removing this + its PDF altogether.